### PR TITLE
Roll NPM Dependencies Forward

### PR DESCRIPTION
**Pull request by JT2M0L3Y**
## Changes
* discord.js [v14.16.2](https://github.com/discordjs/discord.js/releases/tag/14.16.2) added a type-guard for text-based channels to check if messages can be sent: [#10482](https://github.com/discordjs/discord.js/pull/10482)
* updated @types/node, @vitest/coverage-v8, vitest, and tsx with their latest respective versions

### CODE REVIEW

File: src/utils/messageEmbed.ts [hunk 1]
```diff
@@ -88,19 +88,19 @@ export async function embedMessage(
                     const whileEmbed = new EmbedBuilder()
                     .setTitle(`Responding to ${message.author.tag}`)
                     .setDescription(result.slice(0, 5000) || 'No Content to Provide...')
-                    .setColor('#00FF00')
+                    .setColor('#00FF00');
```
Comment (kevinthedang) [inline src/utils/messageEmbed.ts:91]: Don't need a semicolon here.

**Review by kevinthedang — CHANGES_REQUESTED** [review]

File: src/utils/messageEmbed.ts (hunk 1 above)
Comment (JT2M0L3Y) [reply src/utils/messageEmbed.ts:91]: In this case, a semicolon does seem to be needed in order to allow the channel to be casted as a `SendableChannels` type.

Comment (JT2M0L3Y) [reply src/utils/messageEmbed.ts:91]: I can try and find a better way to cast the channel as the `SendableChannels` type... maybe that means climbing up the inheritance tree a few rungs.

Comment (kevinthedang) [reply src/utils/messageEmbed.ts:91]: Gonna investigate as well.

Comment (kevinthedang) [reply src/utils/messageEmbed.ts:91]: Found a solution. Going to implement and push when done.

**Review by kevinthedang — APPROVED** [review]

**Comment by JT2M0L3Y** [comment]
Ooo, this is elevated to a fix pack...

**Pull request merged**